src/SWP/TemplateEngineBundle/Model/ContainerData.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Template Engine Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\TemplateEngineBundle\Model;
+
+use SWP\TemplateEngineBundle\Model\Container;
+
+/**
+ * ContainerData.
+ */
+class ContainerData
```

Comment:
should this class implement an interface? as the implementation will depend from the concrete class. Having an interface will give you more (again) flexibility. you would be able to resolve the entity based on interface